consensus/src/chained_bft/persistent_liveness_storage.rs

```diff
@@ -147,14 +149,39 @@ impl LedgerRecoveryData {
     }
 }
 
+pub struct RootMetaData {
```

Comment:
Metadata

Reply:
This may very well end up in consensus types, since we'll need to keep track of the latest root hash there too...